src/components/util/NavLink/NavLink.tsx

```diff
@@ -1,27 +1,29 @@
 // Inspired by https://stackoverflow.com/questions/60324081/adding-active-class-for-nav-link-in-next-js
 import React from 'react'
 import { useRouter } from 'next/router'
+import Link, { LinkProps } from 'next/link'
 import classnames from 'classnames'
 import { pathToRegexp } from 'path-to-regexp'
 
-import LinkTo from '../LinkTo/LinkTo'
-import type { PropTypes as LinkToProps } from '../LinkTo/LinkTo'
-
-export type NavLinkProps = LinkToProps & {
+export type NavLinkProps = LinkProps & {
+  children: React.ReactNode
+  className?: string
   exact?: boolean
   activeClass?: string
 }
 
 const NavLink = ({
+  children,
   className,
   exact,
   activeClass = 'usa-current',
   ...otherProps
 }: NavLinkProps) => {
-  const { href, as } = otherProps
+  const { href } = otherProps
   const { asPath } = useRouter()
 
-  const pathProp = as || href
+  const pathProp = href
+
```

Comment:
I removed the `as` value here because it was not being evaluated properly. Also, I could not find an instance of it being used as a prop. Let me know if I'm missing something here!